enterprise/management/commands/bulk_update_catalog_query_id.py

```diff
@@ -27,13 +27,15 @@ class Command(BaseCommand):
 
     def add_arguments(self, parser):
         parser.add_argument(
-            'old_id',
-            action='store',
+            '-o', '--old_id',
+            metavar='OLD_ID',
+            dest='old_id',
             help='Old catalog query ID to be replaced.'
         )
         parser.add_argument(
-            'new_id',
-            action='store',
+            '-n', '--new_id',
+            metavar='NEW_ID',
+            dest='new_id',
```

Comment:
do you not need the action anymore?

Reply:
The command I'm going off of didn't use action. I also believe `store` is default?

Reply:
from the docs: `'store' - This just stores the argument’s value. This is the default action`